README.md

```diff
@@ -38,4 +38,34 @@ And in another terminal run the client:
 $ npm run dev
 ```
 
-Now visit the page `http://localhost:8080/`
\ No newline at end of file
+Now visit the page `http://localhost:8080/`
+
+# How to run deploy through bothub
+
+First install bothub-cli
+```
+$ pip install bothub-cli
+```
+Be sure to get the needed credentials and  put it in the .bothub folder
+```
+mkdir .bothub
+cd C:\Users\User\.bothub
```

Comment:
don't need this for the `bothub channel add`

Reply:
Removed it, I figured I'd navigate the person to the spot to store the credentials